Vagrantfile

```diff
@@ -0,0 +1,31 @@
+Vagrant.configure(2) do |config|
+
+  config.vm.box = "trusty-cloud-image"
```

Comment:
This line got added to the cookiecutter: https://github.com/reddit/baseplate-cookiecutter/blob/master/baseplate_cookiecutter/%7B%7Bcookiecutter.project_slug%7D%7D/Vagrantfile#L3

It's necessary for the vagrantfile to work when the user hasn't already cached that box from another vagrantfile.